pkgs/top-level/all-packages.nix

```
@@ -571,6 +571,8 @@ with pkgs;
 
   eclipse-mat = callPackage ../development/tools/eclipse-mat { };
 
+  ebpf-verifier = callPackage ../tools/networking/ebpf-verifier { };
```

```suggestion
  ebpf-verifier = callPackage ../tools/networking/ebpf-verifier { 
    # Replace this to `catch2 = catch2_3` when catch2 3.4.0 is merged
    catch2.src = fetchFromGitHub {
      owner = "catchorg";
      repo = "Catch2";
      rev = "ac93f1943762f6fc92f0dc5bac0d720a33a27530";
      hash = "sha256-2N+PJEfU3A2HY4AARR+PHiENTSlvzlJoObDARDJl15A=";
    };
  };
```

P.S.: shouldn't you use rev = [v3.4.0](https://github.com/catchorg/Catch2/releases/tag/v3.4.0) instead?

P.P.S.: oh yes, I have noticed the patch now. However, if this is compatible then there is no reason not to do this anyway...